napari/components/_tests/test_viewer_keybindings.py

```diff
@@ -0,0 +1,36 @@
+from napari.components._viewer_key_bindings import toggle_theme
+from napari.components.viewer_model import ViewerModel
+from napari.settings import get_settings
+from napari.utils.theme import available_themes, get_system_theme
+
+
+def test_theme_toggle_keybinding():
+    viewer = ViewerModel()
+    assert viewer.theme == get_settings().appearance.theme
```

Comment:
Optional: given the upcoming assertions, you should add an explicit assert a precondition to clarify that `viewer.theme` actually changed later - you can also infer that (since the setting is no longer `light`), but why make the reader do extra logic in their head?

```suggestion
    assert viewer.theme == get_settings().appearance.theme
    assert not viewer.theme == 'light'
```